invenio_circulation/pid/fetchers.py

```diff
@@ -13,7 +13,7 @@
 from ..config import _CIRCULATION_LOAN_PID_TYPE
 
 
-def loanid_fetcher(record_uuid, data):
+def loan_pid_fetcher(record_uuid, data):
```

Comment:
Should be done in a different commit.

Reply:
you are right, but is it ok for now to leave it as it is because it's not only that if we want to split it in different commits? If it's not I will do it, no worries!